lib/iris/analysis/stats.py

```diff
@@ -41,15 +42,17 @@ def _get_calc_view(cube_a, cube_b, corr_coords):
     * corr_coords:
         Names of the dimension coordinates over which
         to calculate correlations.
+    * weights (optional):
+        Numpy array of weights of same shape as cube_a and cube_b
 
     Returns:
 
-    * reshaped_a/reshaped_b:
-        The data arrays of cube_a/cube_b reshaped
-        so that the dimensions to be compared are
-        flattened into the 0th dimension of the
-        return array and other dimensions are
-        preserved.
+    * reshaped_a/reshaped_b/reshaped_weights:
+        The data arrays of cube_a/cube_b and weights
```

Comment:
Since you are in here... if you feel like changing the line lengths to a more readable 72-79 characters it would be appreciated! I find the shorter lines harder to read anyway.